app/src/main/java/com/lightricks/feedexercise/data/Factory.kt

```diff
@@ -0,0 +1,38 @@
+package com.lightricks.feedexercise.data
+
+import android.content.Context
+import androidx.room.Room
+import com.lightricks.feedexercise.database.FeedDatabase
+import com.lightricks.feedexercise.network.FeedApiService
+import com.squareup.moshi.Moshi
+import com.squareup.moshi.kotlin.reflect.KotlinJsonAdapterFactory
+import retrofit2.Retrofit
+import retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory
+import retrofit2.converter.moshi.MoshiConverterFactory
+
+class Factory {
+    companion object Factory {
+        fun createDatabase(context: Context): FeedDatabase {
```

Comment:
This is okay for the exercise. But note that in a real app, database creation must be synchronized.